cashu/wallet/cli/cli_helpers.py

```diff
@@ -215,11 +215,7 @@ async def get_mint_wallet(ctx: Context):
     mint_wallet = Wallet(
         mint_url, os.path.join(settings.cashu_dir, ctx.obj["WALLET_NAME"])
     )
-    mint_keysets: WalletKeyset = await get_keyset(mint_url=mint_url, db=mint_wallet.db)  # type: ignore
```

Comment:
This uses a `fetchone` lookup which means that the wallet will load the first keyset_id it will get from the db.